File: 88-MergeSortedArray/88-MergeSortedArray.java
// Last updated: 24/07/2026, 23:54:05
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int k=0;
        for(int i=0;i<m+n;i++){
            if (i>=m){
                nums1[i]=nums2[k];
                k++;
            }
        }
    Arrays.sort(nums1);
    }
}

File: 88-MergeSortedArray/88-MergeSortedArray.py
# Last updated: 24/07/2026, 23:54:11
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        k=0
        for i in range(0,m+n):
            if i>=m:
                nums1[i]=nums2[k]
                k+=1
        list.sort(nums1)
